### DISCUSSION TITLE
Task 7 Q4 - Finding Homography

### DISCUSSION THREAD

**Discussion by rockyx50** [chulminy/CIVE497-CIVE700#134] [Q&A]
I was running my code and I was unable to correctly compute the homography based on the features. When comparing my code to the example code from last year I found that the only difference between the two is that the example report divides every value in the H matrix by the last value of the H matrix such that H33 is 1 (see code below). When I add this to my code it works. What is the purpose of this line?

UPDATE: Turns out I had my H matrix transposed, which is why it didn't work. Not related to that line of code. But I'm interested to know if that line of code does anything?

![image](https://user-images.githubusercontent.com/60040283/113447450-e5e99e80-93c7-11eb-89ef-009a00f16400.png)

**Comment by chulminy** [comment]
No special meeting. "Up to scale" is the property of vector or matrix in the homogenous coordinate.

If you can test with H and H./H(3,3), you will get the same outcomes.
